pkg/catalog/catalog.go

```diff
@@ -87,13 +90,40 @@ func VersionExists(client *occlient.Client, componentType string, componentVersi
 }
 
 // getDefaultBuilderImages returns the default builder images available in the
-// openshift namespace
+// openshift and the current namespaces
 func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
-	imageStreams, err := client.GetImageStreams(occlient.OpenShiftNameSpace)
-	if err != nil {
-		return nil, errors.Wrap(err, "unable to get Image Streams")
+	var imageStreams []imagev1.ImageStream
+	currentNamespace := client.GetCurrentProjectName()
+	var err error
+
+	// Fetch imagestreams from default openshift namespace
+	openshiftNSImageStreams, e := client.GetImageStreams(occlient.OpenShiftNameSpace)
+	if e != nil {
```

Comment:
i feel as though `e` is a bad name for error, can't we just use `err`?

Reply:
Not err because, I am using it as an overarching error in other words to recognize complete failure.
For example, in this case, its possible that a particular imagestream is not found in the namespace that is being searched, in which case, the other possible namespace needs to be searched also to conclude complete failure. So, this means the first failure is not necessarily actual failure. so, to differentiate the 2(complete failure and not/partial failure I needed it to be err and e separated). If I use the same var, the previous error if any, will be overwritten and the previous error will not be available